### PR TITLE
libc/math: Fix violation of code rule

### PR DESCRIPTION
**Pull request by lokeshbv**
> Declare each variable in new line.
> Use correct exp*() api.

Signed-off-by: Lokesh B V <lokesh.bv@partner.samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c64144e20ec5d15c3b39ca0167c0403a6f98b504] - Code Rule Check (C++) OK.

File: lib/libc/math/lib_erf.c [hunk 1]
```diff
@@ -83,6 +88,6 @@ double erf(double x)
 
 	sign = (x >= 0 ? 1 : -1);
 	t = 1.0 / (1.0 + p * x);
-	return sign * (1.0 - (((((a5 * t + a4) * t) + a3) * t + a2) * t + a1) * t * expf(-x * x));
+	return sign * (1.0 - (((((a5 * t + a4) * t) + a3) * t + a2) * t + a1) * t * exp(-x * x));
```
Comment (sunghan-chang) [inline lib/libc/math/lib_erf.c]: Could you split these kind change from this commit?

**Comment by sunghan-chang** [comment]
After checking compilation from travis, I will merge this. Thank you for contribution.

**Comment by seinfra** [comment]
Target : [8b7c2d76127a35fd2631c76ed76124a084a5458a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8b7c2d76127a35fd2631c76ed76124a084a5458a] - Code Rule Check (C++) OK.

**Pull request merged**